ecosystem/indexer-grpc/indexer-grpc-utils/src/file_store_operator/gcs.rs

```diff
@@ -252,16 +298,34 @@ impl FileStoreOperator for GcsFileStoreOperator {
             anyhow::bail!("Uploading transactions failed.");
         }
 
+        tracing::info!(
+            start_version = start_version,
+            duration_in_secs = start_time.elapsed().as_secs_f64(),
+            service_type = SERVICE_TYPE,
+            protobuf_serialized_size = protobuf_serialized_size,
+            uploaded_size = uploaded_size,
+            batch_size = batch_size,
+            serialization_duration_in_secs = serialization_start_time.elapsed().as_secs_f64(),
+            "Uploaded data to GCS.",
```

Comment:
use log_grpc_step and add a new step for "Uploaded data to GCS" in counters.rs for this